operator/controller/src/test/java/io/apicurio/registry/operator/it/SmokeITTest.java

```diff
@@ -40,5 +44,16 @@ void demoDeployment() {
                     .getSpec().getClusterIP()).isNotBlank();
             return true;
         });
+
+        // Ingresses
+        await().ignoreExceptions().until(() -> {
```

Comment:
Since now, we have access to both the API and the UI, can you please, add an integration test to access those?